doc/source/getting-started/qibosoq_lab_cal.rst

```diff
@@ -0,0 +1,486 @@
+
+
+File ``platform.yml``:
+
+.. code-block:: yaml
+
+  nqubits: 1
+  qubits: [0]
+  topology: []
+  settings: {nshots: 1024, relaxation_time: 70000, sampling_rate: 9830400000}
+
+  native_gates:
+      single_qubit:
+          0:
+              RX:  # pi-pulse for X gate
+                  duration: 40
+                  amplitude: 0.5
+                  frequency: 5_500_000_000
+                  shape: Gaussian(3)
+                  type: qd
+                  start: 0
+                  phase: 0
+
+              MZ:  # measurement pulse
+                  duration: 2000
+                  amplitude: 0.02
+                  frequency: 7_370_000_000
+                  shape: Rectangular()
+                  type: ro
+                  start: 0
+                  phase: 0
+
+      two_qubits: {}
+  characterization:
+      single_qubit:
+          0:
+              readout_frequency: 7370000000
+              drive_frequency: 5500000000
+              anharmonicity: 0
+              Ec: 0
+              Ej: 0
+              g: 0
+              T1: 0.0
+              T2: 0.0
+              threshold: 0.0
+              iq_angle: 0.0
+              mean_gnd_states: [0.0, 0.0]
+              mean_exc_states: [0.0, 0.0]
+
+
+Every experiment, will then start with:
+
+.. code-block:: python
+
+  from qibolab import create_platform
+  from qibolab import AcquisitionType, AveragingMode, ExecutionParameters
+
+  from qibolab.pulses import (
+      DrivePulse,
+      ReadoutPulse,
+      PulseSequence,
+  )
+
+  # Define platform and load specific runcard
+  platform = create_platform("platform")
+
+Qibocal
+-------
+
+For Qibocal, we first need to setup Qibolab as presented in the last section.
+Note then that, for Qibocal "programs" we need a new file ``actions.yml`` that will contain all the parameters required for the experiments: this file will be presented for all the different experiments.
+For Qibosoq and Qibolab "programs", a standard Python script or a Jupyter Notebook will suffice.
+
+
+Time Of Flight
+""""""""""""""
+
+Qibosoq
+-------
+
+.. code-block:: python
+
+  from qibosoq.components.pulses import Rectangular
+
+  pulse = Rectangular(
+            frequency = 7400, #MHz
+            amplitude = 0.5,
+            relative_phase = 0,
+            start_delay = 0,
+            duration = 1,
+            name = "readout_pulse",
+            type = "readout",
+            dac = 1,
+            adc = 0
+  )
+
+  sequence = [pulse]
+  config = Config(
+            repetition_duration=0.05,
+            adc_trig_offset=0,
+            reps=1,
+            soft_avgs=1000,
+            average=False
+  )
+  qubit = Qubit()
+
+  server_commands = {
+      "operation_code": OperationCode.EXECUTE_PULSE_SEQUENCE_RAW,
+      "cfg": config,
+      "sequence": sequence,
+      "qubits": [qubit],
+  }
+
+  i, q = execute(server_commands, HOST, PORT)
+
+  plt.plot(np.abs(np.array(i) + 1j * np.array(q))
+
+
+Qibolab
+-------
+
+.. code-block:: python
```

Comment:
Same for Qibolab: it is fine to have it written above, but I would actually repeat it over and over.